ui/client/components/graph/node-modal/editors/field/Field.js

```diff
@@ -5,33 +5,20 @@ import Checkbox from "./Checkbox"
 import UnknownField from "./UnknownField"
 import Input from "./Input"
 
-const FieldTypes = {
-  LabeledInput,
-  LabeledTextarea,
-  Checkbox,
-  UnknownField,
-  Input
-}
-
-const resolveComponentType = type => {
-  switch (type) {
+export const Field = (props) => {
+  const {fieldType} = props
+  switch (fieldType) {
     case 'input':
-      return LabeledInput.name
+      return (<LabeledInput {...props}/>)
     case 'unlabeled-input':
-      return Input.name
```

Comment:
Did you tested it with just using "LabeledInput", "Input", "Checkbox" and so on? Maybe we still can use "Capitalized Reference Technique" (https://medium.com/@j5bot/react-dynamically-rendering-different-components-without-switch-the-capitalized-reference-e668d89e460b) ?